enable/trait_defs/ui/kiva_font_editor.py

```diff
@@ -23,6 +23,20 @@
 from .editor_with_component import EditorWithLabelComponent
 
 
+WEIGHTS = {
+    kc.WEIGHT_THIN: ' Thin',
+    kc.WEIGHT_EXTRALIGHT: ' Extra-light',
+    kc.WEIGHT_LIGHT: ' Light',
+    kc.WEIGHT_NORMAL: '',
+    kc.WEIGHT_MEDIUM: ' Medium',
+    kc.WEIGHT_SEMIBOLD: ' Demi-bold',
```

Comment:
Font question, possibly off-topic for this PR: I don't think I've met the term "Demi-bold" before. Is it completely interchangeable with semi-bold, and is there a reason to prefer the "Demi-bold" description over "Semi-bold"? The mix of the two terms feels confusing.

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#common_weight_name_mapping seems to support your preference